pkg/pillar/types/downloadertypes.go

```diff
@@ -167,3 +168,71 @@ func AllowNonFreePort(gc ConfigItemValueMap, objType string) bool {
 		return false
 	}
 }
+
+// ResolveConfig key/index to this is the ImageID which is allocated by the controller.
+// It will resolve container image tag to sha256
+type ResolveConfig struct {
+	AppID            uuid.UUID
+	ImageID          uuid.UUID
+	DatastoreID      uuid.UUID
+	Name             string
+	NameIsURL        bool // If not we form URL based on datastore info
```

Comment:
I don't expect this to be ever used when resolving. (Sorry, didn't see it earlier.)
It is only used by the certObj today in downloader. So you can drop it from here.

Reply:
Removed.